drivers/spi/Kconfig.xec_qmspi_v2_ldma

```diff
@@ -0,0 +1,18 @@
+# Microchip XEC QMSPI V2, QMSPI LDMA
+
+# Copyright (c) 2022 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+choice
```

Comment:
Why the choice?  Isn't this determined by what nodes are in devicetree with which compatibles?

Reply:
the choice was added based on review comments in this PR.
we have two flavor of driver for MEC172x qmspi and depending on the application use case we want to have a choice to pick the right version.